crates/solver/src/solver/baseline_solver.rs

```diff
@@ -609,7 +609,7 @@ mod tests {
             sell_amount: 1_741_103_528_769_588_955_u128.into(),
             buy_amount: 500_000_000_000_000_000_000_u128.into(),
             kind: OrderKind::Buy,
-            id: "Crash Bandicoot".to_string(),
```

Comment:
Sorry @nlordell , "Crash Bandicoot" had to go :)

Reply:
🤣